README.md

```diff
@@ -380,6 +382,27 @@ This section describes what you need to change to make your code compile with `r
    + Use `(putStr[Ln]|readFile|writeFile|appendFile)[Text|LText|BS|LBS]` functions.
 6. Run `hlint` using `.hlint.yaml` file from `relude` package to cleanup code and imports.
 
+### Running HLint on CI
+
+Instead of storing `relude`-specific `.hlint.yaml` file inside your repository,
+you can run HLint with this file automatically on any CI service such as
+[Travis CI](https://travis-ci.org/) or [Circle CI](https://circleci.com/).
+For this you need to:
+
+1. Find commit hash of the `relude` version you are using (can be found in [releases](https://github.com/kowainik/relude/releases)).
+2. Run command that downloads `.hlint.yaml` for that version.
+3. Run `hlint` using this file.
+
+For the latest `relude` version, this can be achieved by executing the following
+two commands on your CI:
+
+```yaml
+curl https://raw.githubusercontent.com/kowainik/relude/988bbdd3a09df1159917012933780523644880e5/.hlint.yaml -o .hlint-relude.yaml
```

Comment:
Keep a little space between these 2 commands and maybe specify the step number before each one. 
Just for absolute clarity (even though it's pretty obvious) 

This is just a suggestion 😄